src/Index.js

```diff
@@ -46,7 +46,7 @@ Index.prototype.addObject = function(content, objectID, callback) {
     hostType: 'write',
     callback: callback
   });
-};
+}, deprecatedMessage('index.addObject()', 'index.addObjects()'));
```

Comment:
perfect, that means that behaviour won't change between v3 and v4 of the plural forms. 

I'm *slightly* wondering if there might be a performance difference between the single form and the plural form with a single item, WDYT (maybe someone from core team?)